sdk/client/commander_test.go

```diff
@@ -206,10 +206,12 @@ func TestCommander_Connect_NoServer(t *testing.T) {
 	commanderClient := NewCommanderClient()
 	commanderClient.WithServer("unknown")
 	commanderClient.WithDialOptions(grpcDialOptions...)
-	commanderClient.WithBackoffSettings(backoff.BackoffSettings{
-		InitialInterval: 100 * time.Millisecond,
-		MaxInterval:     100 * time.Millisecond,
-		MaxElapsedTime:  300 * time.Millisecond,
+	commanderClient.WithProtoBackoffSettings(&proto.Backoff{
```

Comment:
maybe centralize in the test utils GetMockAgentConfig - you can call GetMockAgentConfig().Server.Backoff then?

Reply:
looking at this at the moment, cant use test utils as the import cycle isn't aloud but will do something to not repeat the values